sdcm/cluster_gce.py

```diff
@@ -308,17 +343,16 @@ def to_short_name(value):
 
             normalized_tags = {to_short_name(k): to_short_name(v) for k, v in self.tags.items()}
             self.log.debug(f"normalized tags are {normalized_tags}")
-            self._gce_services[dc_idx].ex_set_node_labels(_instance, normalized_tags)
 
-        # if size of disk is larget than 80G, then
-        # change the timeout of job completion to default * 3.
+            request = compute_v1.InstancesSetLabelsRequest()
+            request.labels = _instance.labels
+            request.label_fingerprint = _instance.label_fingerprint
+            request.labels.update(normalized_tags)
+
+            self._gce_service.set_labels(project=self.project, zone=self._gce_zone_names[dc_idx],
+                                         instance=_instance.name,
+                                         instances_set_labels_request_resource=request)
```

Comment:
this code block is repeated multiple times - is it feasible to extract/reuse one method/function?

Reply:
refactored it into `gce_utils.gce_set_labels`